### PR TITLE
Editor: Add 'New Project', 'Load Project' and 'Save Project'

### PR DESCRIPTION
**Pull request by codedeep**
**Description**

Into Editor change "New" to "New Project" and add "Load Project" and "Save Project" to "File" Menu for higher productivity

Change translation "New" to "New Project" and add "Load Project" and "Save Project". 
French and Chinese translations would be missing.

translations_menu

### CODE REVIEW

**Comment by Mugen87** [comment]
It seems this PR is an alternative implementation of #19441, correct?

**Comment by codedeep** [comment]
Yes @Mugen87.

**Comment by Mugen87** [comment]
For testing: https://rawcdn.githack.com/codedeep/three.js/patch-1/editor/index.html

File: editor/js/Menubar.File.js [hunk 1]
```diff
@@ -38,21 +38,67 @@ function MenubarFile( editor ) {
...
 	container.add( options );
 
-	// New
+	// New Project
 
 	var option = new UIRow();
-	option.setClass( 'option' );
-	option.setTextContent( strings.getKey( 'menubar/file/new' ) );
-	option.onClick( function () {
+	option.setClass("option");
```
Comment (Mugen87) [inline editor/js/Menubar.File.js:44]: Please honor the code style of the project. E.g. `option.setClass("option");` should be:
```js
option.setClass( 'option' );
````
It's actually best if you configure your IDE/editor so it automatically detects the eslint settings ofthe repository.

Comment (codedeep) [reply editor/js/Menubar.File.js:44]: Sorry, Pettier plugin with bad configuration

File: editor/js/Menubar.File.js [hunk 2]
```diff
@@ -38,19 +38,78 @@ function MenubarFile( editor ) {
...
 
+	var option = new UIRow();
+	option.setClass( 'option' );
+	option.setTextContent( strings.getKey( 'menubar/file/load' ) );
+	option.onClick( function() {
+		
+		if ( confirm( 'Any unsaved data will be lost. Are you sure?' ) ) {
+			
 			editor.clear();
+			openfileInput.click();
```
Comment (Mugen87) [inline editor/js/Menubar.File.js:83]: Um, it feels a bit strange that the editor is already cleared at this point. I guess it's better when the clear should only happen if the user actually selects a JSON from the input dialog.

Comment (codedeep) [reply editor/js/Menubar.File.js:83]: Good observation

File: editor/js/Menubar.File.js [hunk 3]
```diff
@@ -48,8 +48,67 @@ function MenubarFile( editor ) {
...
+	option.setClass( 'option' );
+	option.setTextContent( strings.getKey( 'menubar/file/save' ) );
+	option.onClick(function () {
+
+		var output = editor.toJSON();
+		output.metadata.type = 'App';
+
+		try {
+			output = JSON.stringify( output, parseNumber, '\t' );
+			output = output.replace( /[\n\t]+([\d\.e\-\[\]]+)/g, '$1' );
```
Comment (Mugen87) [inline editor/js/Menubar.File.js:101]: What is the purpose of this line?

**Comment by Mugen87** [comment]
Closing due to lack of feedback.

On second glance, the `Save Project` menu item is strange since saving the project is already possible (that happens automatically by default). `Export Project` as suggested in #19441 sounds less confusing.

**Pull request closed without merging**